pkg/skaffold/deploy/util.go

```diff
@@ -70,12 +72,19 @@ func parseReleaseInfo(namespace string, b *bufio.Reader) []Artifact {
 			logrus.Infof("error parsing object from string: %s", err.Error())
 			continue
 		}
+
+		// The initial section of "helm get" is not a Kubernetes manifest
+		if i == 1 {
+			continue
```

Comment:
Is there any value in logging that section header?

I'm (very mildly) worried that Helm 4 or 5 might starting emitting  valid k8s manifests and we'll throw this first element away.  Would it make sense to instead try parsing it and instead ignore the error if it's the first object?